app/agents/voice/automatic/tools/system/utils.py

```
@@ -12,7 +12,8 @@ async def get_current_time(params: FunctionCallParams):
         current_time = datetime.now(tz).isoformat()
         await params.result_callback({"time": current_time})
     except Exception as e:
-        await params.result_callback({"error": str(e)})
+        logger.error(f"Tool Error: [get_current_time] Error getting current time: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)